README.md

```diff
@@ -29,81 +29,34 @@ The Plaid Postman collection uses [Postman environment variables](https://learni
-  * **Remove Asset Report** - Removes an Asset Report.
-  * **Remove Audit Copy** - Removes an Audit Copy.
-  * **Refresh Asset Report** - Create a new Asset Report based on the old one, but with the most recent data available from the financial institution(s).
+## Making Postman calls with real data in Production or Development
 
-* **Institutions**
-  * **Search Institution by Name** - Retrieves information about a specific institution by name.
-  * **Search Institution by ID** - Retrieves information about a specific institution by ID.
-  * **Retrieve Institution List** - Retrieves a list of all supported institutions.
+For reasons of security and transparency, getting an access token for use with real data cannot be done entirely via Postman API calls -- you are required to use the Plaid Link UI component. You can do this as follows:
 
-* **Categories**
-  * **Retrieve Categories** - Retrieves detailed information on categories returned by Plaid. This endpoint does not require authentication.
+1. Download the [link.html](/link.html) file included in this repo and open it in a text editor (alternatively, open a text editor, create a new file called link.html, and copy and paste the contents of [link.html](/link.html) into it). You will use this file later.
+2. Re-visit the "Configuration" steps at the top of this page, but instead change the `client_id` and `secret_key` environment variables to your client ID and secret for Production (or Development) instead of for Sandbox, then set the `env_url` environment variable to `production.plaid.com` (for Production) or `development.plaid.com` (for Development).
+3. Navigate to Plaid API Endpoints -> Link Tokens -> Create Link Token and click on the "Body" tab.
+4. If you want, replace "auth" with the name of the product you would like to try, such as "transactions" or "identity". Note that only institutions that support ALL the products you specify here will appear in Link -- if you don't see the institution you want when Link is launched, make sure you have listed the right product. For example, an institution that only supports credit cards (e.g. American Express) will not appear in Link if a product that doesn't support credit cards (like auth) is specified.
+5. Click the blue "Send" button.
+6. Copy the value of the `link_token` from the response.
+7. Return to your local copy of link.html. Replace the text 'your-link-token-goes-here' in link.html with the value you copied in the previous step, and save the file.
+8. Open link.html in a web browser.
+9. Open the web browser Developer Tools (for example, in Chrome, go to View->Developer->Developer Tools), then open the Console tab within Developer tools.
+10. Click the "Link Account" button on link.html to launch Link. The Plaid Link component should launch. Follow the prompts and log into a financial institution.
```

Comment:
Not sure if I did something wrong, but attempting to do this resulted in a "429: Too many requests" error

Reply:
my guess is that you tried to use the sandbox creds instead of production ones. so many people accidentally try to log in to real banks in Link as user_good that it gets heavily rate limited for sketchy-looking login attempts

Reply:
That's a good theory. I'll try this again on my own at a later point.